wgpu-types/src/lib.rs

```diff
@@ -526,6 +526,13 @@ bitflags::bitflags! {
         ///
         /// This is a native only feature.
         const SHADER_PRIMITIVE_INDEX = 1 << 39;
+        /// Enables multiview render passes and `builtin(view_index)` in vertex shaders.
+        ///
+        /// Supported platforms:
+        /// - Vulkan
+        ///
+        /// This is a native only feature.
+        const MULTIVIEW = 1 << 40;
```

Comment:
should this really be called `MULTIVIEW`? we don't have separate viewport and scissors for them (yet). It looks like this is more of `LAYERED_RENDERING` thing.

Reply:
This was discussed and resolved on the matrix chat, so I'll resolve this